packages/pages/src/components/image/image.tsx

```diff
@@ -91,21 +79,43 @@ export const Image = ({
   );
 };
 
-const getImageUrl = (uuid: string, width: number, height: number) => {
+/**
+ * Returns the UUID of an image given its url. Logs a warning message if the image url is invalid.
+ */
+export const getImageUUID = (url: string) => {
```

Comment:
Does this need to be exported?

Reply:
I exported them for testing but they are not exported in index.ts